vwr/driver.py

```diff
@@ -136,9 +159,14 @@ def close(self):
     def _send(self, message):
         """Selds a message to the circulating bath."""
         if self.waiting:
-            raise IOError("Waiting for another bath request to be processed.")
-        self.sender.sendto((message + '\r').encode('utf-8'),
-                           (self.address, 1024))
+            Timer(5, self._send, [message]).start()
```

Comment:
I don't know if this will work, but we can try it on the bath and see what happens. However, I'd take 5s down to <=1s. Running this function is not expensive if it short-circuits on `if self.waiting`.

Also, you probably need to wrap everything below in an `else` block, otherwise this function will still send.